rustls/src/msgs/message/mod.rs

```diff
@@ -44,6 +48,10 @@ impl<'a> MessagePayload<'a> {
         }
     }
 
+    pub(crate) fn handshake_flight(encoded: Vec<u8>) -> Self {
```

Comment:
nit: visibility is sub `pub` and so should be lower in order.

Reply:
Given only one library caller and one test caller, maybe we don't need this at all?

Reply:
Dropped.